src/webhooks/options-automator/options-automator.ts

```diff
@@ -0,0 +1,137 @@
+import * as Sentry from '@sentry/node';
+import { KnownBlock } from '@slack/types';
+import { FastifyRequest } from 'fastify';
+
+import { OptionsAutomatorResponse } from '@types';
+
+import { bolt } from '@/api/slack';
+import * as slackblocks from '@/blocks/slackBlocks';
+import { FEED_OPTIONS_AUTOMATOR_CHANNEL_ID } from '@/config';
+
+export async function handler(
+  request: FastifyRequest<{ Body: OptionsAutomatorResponse }>
+) {
+  const { body }: { body: OptionsAutomatorResponse } = request;
+  await messageSlack(body);
+  return {};
+}
+
+export async function messageSlack(message: OptionsAutomatorResponse) {
+    const successBlock: KnownBlock[] = [
+        slackblocks.header(slackblocks.plaintext('✅ Successfully Updated Options: ✅')),
+        ...(message.channel_updated_options.length > 0
+            ? [
+                slackblocks.divider(),
+                slackblocks.section(slackblocks.markdown('*Channel updated options:* ')),
+                slackblocks.SectionBlock(
+                message.channel_updated_options.map((option) =>
+                    slackblocks.markdown(`channel updated \`${option}\``)
+                )
```

Comment:
Channel update is not a useful information for the users. Please remove them entirely.
We only care about successful and failed changes in the option value.